birdhouse/deployment/deploy.sh

```diff
@@ -66,7 +61,7 @@ COMPOSE_DIR="$1"
 ENV_LOCAL_FILE="$2"
 
 if [ -z "${COMPOSE_DIR}" ]; then
-    echo "ERROR: please provide path to PAVICS docker-compose dir." 1>&2
+    echo "ERROR: please provide path to Birdhouse docker-compose dir." 1>&2
```

Comment:
Actually this reminds me ... 

@tlvu can you advise how to make use of the `BIRDHOUSE_COMPOSE` variable in this file. I remember that it can be called in multiple contexts, one of them is during autodeploy, what is the other?

Reply:
Well `COMPOSE_DIR` variable is used for many sauces:

* load all the `docker-compose-extra.yml`, `default.env`, `read-configs.include.sh` which load `logging.include.sh`

* set the `PWD` when invoking `pavics-compose.sh` via `deploy.sh` (autodeploy) or via any other external scripts from any other external repos.

It's pretty much the anchor point of many pieces so they can reference each other without having to know each other physical location.  All references are pretty much relative to that `COMPOSE_DIR` variable.

Hope I have answered your question.

Reply:
@tlvu sorry I could have phrased that question better. I meant, does this script only run in a docker container deployed by the scheduler or is this script meant to be run manually on the host machine as well?

Reply:
Sorry for the late reply, the script `deployment/deploy.sh` is mostly meant to be run inside the autodeploy container.

Reply:
Thanks. As I mentioned somewhere below I've tested this now with the autodeploy procedure so the changes made to this file should be ok.